### DISCUSSION TITLE
Automatic duplicate issue detection?

### DISCUSSION THREAD

**Discussion by bigpresh** [GriefPrevention/GriefPrevention#1715] [Ideas]
We are getting quite a few duplicate issues raised lately (especially so when a new MC release hits and we're not yet updated for it), and it takes a bit of time to dig out the older issues the new one is a dupe of in order to link to them.

I wonder if we should consider using one of the various duplicate issues detection tools, e.g.

https://github.com/probot/duplicate-issues
https://github.com/Bartozzz/potential-duplicates-bot

They could both help users to locate the issues their question may be a duplicate of, and help save us time triaging the issue and digging out the previous issue(s) it duplicates to mark it up as such and close it.

Thoughts?

**Comment by Jikoo** [comment]
Looks like https://github.com/wow-actions/potential-duplicates is an Actions-based runner similar to Bartozzz's implementation. There's been a heck of a lot of triage lately, definitely wouldn't mind a reduced workload on that front.
